package.json

```diff
@@ -20,6 +20,7 @@
     "publish:npm": "TARGET_DATACENTER=us BUILD_MODE=release yarn build && lerna publish from-package",
     "test": "yarn test:unit",
     "test:unit": "karma start ./test/unit/karma.local.conf.js",
+    "test:unit:watch": "yarn test:unit --no-single-run",
```

Comment:
I also wanted to open a PR for something similar from some time.
What would you think of having this behavior for `yarn test` directly?

Reply:
It is somewhat unusual to have this as the default test command, so as a newcomer I wouldn't expect this behavior. But if it is discoverable as a separate test as well, I wouldn't mind :)

Reply:
So, keep `test:unit:watch` and use it from `test`?